### PR TITLE
Final code review sp

### PR DESCRIPTION
**Pull request by stacyp2006**
#### What's this PR do?

- Cleans up the CSS, HTML and JS files so they are consistent with the respective style guides. 

#### Where should the reviewer start?

- The reviewer can check each file: index.html, styles.css, idea.js and main.js. 

#### How should this be manually tested?

- None of these changes affect functionality, but tester could read through the code. 

#### Any background context you want to provide?

- Style Guide for CSS: https://github.com/turingschool-examples/css
- Style Guide for HTML: https://github.com/turingschool-examples/html
- Style Guide for JavaScript: https://github.com/turingschool-examples/javascript/tree/main/es5

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -148,7 +147,7 @@ body {
 
 .menu-header-icon:hover {
   cursor: pointer;
-  opacity: 80%
+  opacity: 80%;
```
Comment (CalebCyphers) [inline styles.css:158]: Thanks for catching this! I'm really bad with these

File: styles.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/*=== Reset ===*/
+
```
Comment (CalebCyphers) [inline styles.css:2]: This is perfect! thanks for catching this in the style guide! You rock!

**Review by MichaelEWalker87 — APPROVED** [review]
I see that we have updated our file to be up too date with the style guide

**Review by CalebCyphers — APPROVED** [review]
This is looking perfect. You caught several things I wouldn't have, good eye.

**Pull request merged**